packages/angular/cli/upgrade/version.ts

```diff
@@ -61,6 +65,27 @@ export class Version {
       process.exit(2);
     }
 
+    try {
+      const angularCliPkgPath = require.resolve('@angular/cli/package.json', resolveOptions);
```

Comment:
Since we are already in the cli package, should there be an easier (and more efficient) method to retrieve the version?  (e.g., static version field that is set at build time)

Reply:
Unsure. I mean, it's nice and all, but worth adding? I don't think we access this version in many places. Might also complicate things with some global/local setups. 

If we decide to do this, we'd need to add tooling to automatically update it. The bazel npm package toolchain does that I think, so maybe waiting for that would a good approach.

Reply:
sounds like a plan.